build/resnet50/build.py

```diff
@@ -47,7 +50,30 @@
 synth_clk_period_ns = 4.0
 target_fps = 300
 
+verif_en = os.getenv("VERIFICATION_EN", "0")
+
+# which platforms to build the networks for
+zynq_platforms = []
+alveo_platforms = ["U250"]
+platforms_to_build = zynq_platforms + alveo_platforms
+
+model_file = "models/%s_exported.onnx" % model_name
+# create a release dir, used for finn-examples release packaging
+os.makedirs("release", exist_ok=True)
+
+
+def custom_step_update_model(model, cfg):
```

Comment:
Same as above, since we updated the onnx file to reflect a more recent onnx opset version, this update function should be removed.

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)